src/views/LogIn.vue

```diff
@@ -0,0 +1,28 @@
+<template lang="pug">
+form-component(:fieldsArray='fieldsArray' :handleAction='handleLogin' :formData='userData' )
```

Comment:
```suggestion
form-component(:fields-array='fieldsArray' :handle-action='handleLogin' :form-data='userData' )
```

Use kebab-case notation for all props and attributes in the template